### DISCUSSION TITLE
Does Prisma support select with "with (nolock)"?

### DISCUSSION THREAD

**Discussion by renemelo** [prisma/prisma#14288] [Q&A] [answered]
How to use "with (nolock)", usually used in select in SQL Server in Prisma?

**Comment by ruheni** [comment, marked as answer]
Hey 👋🏽

Prisma doesn't currently support this. One workaround before we support this would be using a raw query.

Could you create a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) for this? 🙂

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
